examples/DGmethods/ex_001_dycoms.jl

```diff
@@ -0,0 +1,233 @@
+using MPI
+using CLIMA
+using CLIMA.Mesh.Topologies
+using CLIMA.Mesh.Grids
+using CLIMA.DGmethods
+using CLIMA.DGmethods.NumericalFluxes
+using CLIMA.MPIStateArrays
+using CLIMA.LowStorageRungeKuttaMethod
+using CLIMA.ODESolvers
+using CLIMA.GenericCallbacks
+using CLIMA.Atmos
+using CLIMA.VariableTemplates
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+using LinearAlgebra
+using StaticArrays
+using Logging, Printf, Dates
+using CLIMA.VTK
+
+@static if haspkg("CuArrays")
+  using CUDAdrv
+  using CUDAnative
+  using CuArrays
+  CuArrays.allowscalar(false)
+  const ArrayType = CuArray 
+else
+  const ArrayType = Array 
+end
+
+if !@isdefined integration_testing
+  const integration_testing =
+    parse(Bool, lowercase(get(ENV,"JULIA_CLIMA_INTEGRATION_TESTING","false")))
+end
+
+"""
+  Initial Condition for DYCOMS_RF01 LES
+"""
+function Initialise_DYCOMS!(state::Vars, aux::Vars, (x,y,z), t)
+    
```

Comment:
Based on my conversation with @asraero yesterday, this initialization returns a q_l field with max(q_l)=0.37 instead of the 0.45 requested by Steven's paper. 
To achieve that, the initialization should follow this:

https://github.com/smarras79/CLIMA/blob/02c6dc286b080d71d56fbadd7bd20e1e4dc74237/test/DGmethods/compressible_Navier_Stokes/dycoms-analyticinit.jl#L749